### ISSUE TITLE
Roblox blocking Wine again

### ISSUE THREAD

**Issue by jmfergeau** [vinegarhq/vinegar#404]
As title says. Just like last year. Vinegar launches but then the Roblox player shows this:

![Wine is not supported](https://github.com/vinegarhq/vinegarhq.github.io/assets/6705075/cd3c7efc-ca2e-418c-9c17-309d3f5f00fb)

Here's the logs I got with it:

```
2024/03/02 13:13:17 INFO Fetching Binary Deployment name=WindowsPlayer channel=""
2024/03/02 13:13:17 INFO Binary is up to date! name=WindowsPlayer guid=version-bca459bcd1854ce4
2024/03/02 13:13:17 INFO Enabling WINE DXVK DLL overrides
2024/03/02 13:13:17 INFO Connecting to Discord RPC
2024/03/02 13:13:17 ERROR Could not connect to Discord RPC error="drpc: discord is not running"
2024/03/02 13:13:17 INFO Running Binary name=WindowsPlayer cmd="/app/bin/wine64 /home/max/.var/app/org.vinegarhq.Vinegar/data/vinegar/versions/version-bca459bcd1854ce4/RobloxPlayerBeta.exe -app"
002c:fixme:winediag:loader_init wine-staging 9.2 is a testing version containing experimental patches.
002c:fixme:winediag:loader_init Please mention your exact version when filing bug reports on winehq.org.
2024/03/02 13:13:23 ERROR Failed to find Roblox log file error="roblox log file not found after 6s"
008c:err:sync:RtlpWaitForCriticalSection section 00006FFFFFFB04C0 "dlls/ntdll/loader.c: loader_section" wait timed out in thread 008c, blocked by 0080, retrying (60 sec)
0140:fixme:ntdll:NtRaiseHardError 50000018 stub
0140:fixme:dialog:MessageBoxTimeoutW timeout not supported (0)
2024/03/02 13:13:32 ERROR failed to run roblox: roblox process: exit status 187
2024/03/02 13:13:37 Opening log file: /home/max/.var/app/org.vinegarhq.Vinegar/cache/vinegar/logs/Player-2024-03-02T13:13:16+01:00.log
```

Arch Linux with LTS kernel, using the flatpak stable version of Vinegar.

**Comment by apprehensions** [comment]
Read the issue template, observe existing issues.

Duplicate of #397
